applications/HDF5Application/custom_io/hdf5_file.h

```diff
@@ -376,18 +376,18 @@ void File::ReadAttribute(std::string ObjectPath, std::string Name, TScalar& rVal
     KRATOS_ERROR_IF((ndims = H5Sget_simple_extent_ndims(space_id)) < 0)
         << "H5Sget_simple_extent_ndims failed." << std::endl;
     KRATOS_ERROR_IF(H5Sclose(space_id) < 0) << "H5Sclose failed." << std::endl;
-    KRATOS_ERROR_IF(ndims != 0) << "Attribute \"" << Name << "\" is not scalar." << std::endl;
+    KRATOS_ERROR_IF(ndims != 0) << "Attribute \"" << rName << "\" is not scalar." << std::endl;
```

Comment:
`KRATOS_ERROR_IF_NOT(ndims == 0)`

Reply:
I would leave. As we discussed above, it is basically a style/preference thing.